docker/go-control-plane/fallback-control-plane.go

```diff
@@ -52,12 +48,11 @@ import (
 )
 
 var (
-	port     = flag.Uint("port", 3333, "Port to listen on")
-	nodeid   = flag.String("nodeid", "test-id", "Node ID")
-	upstream = flag.String("upstream", "localhost:3000", "upstream server")
+	port   = flag.Uint("port", 3333, "Port to listen on")
+	nodeid = flag.String("nodeid", "test-id", "Node ID")
 )
 
-type Filter struct {
+type resourceKey struct {
 	ResourceType string
 	ResourceName string
```

Comment:
Missed in the last PR maybe. Any reason for these fields to be exported? If not, please unexport them.